package.json

```diff
@@ -46,6 +46,7 @@
     "normalize-node-version": "^11.0.0",
     "pacote": "^13.0.0",
     "semver": "^7.0.0",
+    "tsx": "^3.9.0",
```

Comment:
This has already been added in #820 but until that's merged, I need it in this PR.